packages/sdk-components-react/src/code-text.ws.ts

```diff
@@ -49,6 +49,13 @@ export const meta: WsComponentMeta = {
 };
 
 export const propsMeta: WsComponentPropsMeta = {
-  props,
-  initialProps: ["id", "className", "lang"],
+  props: {
+    ...props,
+    code: {
+      required: true,
+      control: "code",
```

Comment:
This will highlight it only as html

Reply:
what should we do about it?

Reply:
I see 2 options:
1. use no syntax highlighting at all in this case
2. allow user to select language in the editor

Reply:
technically we already have a lang prop there, we could pick a default and use the same prop to highlight in the editor as well as on the canvas later

The questions there:
1. how do we provide to the code editor the selected langage on the other prop, currently each prop doens't have access to other props
2. we will need to lazy load language packages to avoid loading all of them at once, right?

Reply:
or we keep it as a follow-up issue and for now just have a way to define code editor without syntax highlighting, this will be more appropriate for this issue, which is only trying to prevent user from addding children from this component